README.md

```diff
@@ -18,25 +18,84 @@ Then to serve the site, you have two options:
 
 1. To serve the static (built) version of the website (good for final review):
 
-    ```shell
-    yarn serve
-    ```
+```shell
+yarn serve
+```
 
 2. To serve the site using the development server with auto-compiling and page reloading:
 
-    ```shell
-    yarn start
-    ```
+```shell
+yarn start
+```
+
+## Publishing a blog post
+
+We want and encourage everyone at Sourcegraph to contribute to the blog and publishing a blog post is as simple as merging an approved pull request with your blog content.
+
+The process for publishing a blog post is:
+
+1. Create the blog post file
+1. Create a pull request
+1. Confirm via the preview deployment site that everything is looking good (the `deploy/netlify — Deploy preview ready!` link in the checks section)
+1. Get your pull request approved n(preferably by someone on the marketing team)
```

Comment:
```suggestion
1. Get your pull request approved (preferably by someone on the marketing team)
```

You could create a marketing team on GitHub and then assign them as CODEOWNERS so they are automatically added as reviewers for new blog posts.